src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/documentcatergory/HearingNoticesCategoriserTest.java

```diff
@@ -14,32 +12,33 @@
 
 import java.util.List;
 
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
-public class HearingNoticesCategoriserTest {
+class HearingNoticesCategoriserTest {
 
     HearingNoticesCategoriser hearingNoticesCategoriser;
     FeatureToggleService featureToggleService;
 
-    @Before
-    public void setUpTest() {
+    @BeforeEach
+    void setUp() {
         featureToggleService = mock(FeatureToggleService.class);
         when(featureToggleService.isCaseFileViewEnabled()).thenReturn(true);
         hearingNoticesCategoriser = new HearingNoticesCategoriser(featureToggleService);
     }
 
     @Test
-    public void categoriseDocuments() {
+    void categoriseDocuments() {
```

Comment:
Should this test method be more descriptive? Something specific like:

```suggestion
    void categoriseDocuments_shouldAssignCorrectCategoryToHearingNoticeDocuments() {
```

Reply:
I'd settle for changing it to
`testCategorise`
as categorise is the method being tested.

Reply:
Thanks, have actioned